src/checks/kubernetes/services/service_test.go

```diff
@@ -137,6 +137,36 @@ func Test_inputs_GeneralCheck(t *testing.T) {
 				Message: "* ClusterIP Found\n",
 			},
 		},
+		{
+			name: "Checking clusterIP (negative)",
+			fields: fields{
+				checkName:  "check1",
+				namespace:  "ns1",
+				valuesYaml: "values:\n  serviceName: service1\n  port: 20014\n  checksEnabled:\n    clusterIP: true",
```

Comment:
Is there any method by which you don't have to hardcode the port and can pass it in?

Reply:
This is a unit test...im not passing anything into the unit test.  Maybe i dont understand your question...can you elaborate more on what you mean?

Reply:
You can scratch my concern. I just reviewed the entire file to make sure I fully understood what you were trying to do. Initially I had thought you could create a variable that you could swap out the port numbers in the unit test so you could recycle some of the code blocks.

Reply:
This code block is just values.  We could move this to another file but it would just be a values file with the same content.